src/FixedDataCellGroupFunction.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import FixedDataTableTranslateDOMPosition from './FixedDataTableTranslateDOMPosition';
+import cx from './vendor_upstream/stubs/cx';
+
+function CellGroup(props) {
+  var style = {
```

Comment:
here or every other place, use `const` instead,  or `let` (only  if you have to modify variable later).